tests/test_app_runner.py

```diff
@@ -6,9 +6,23 @@
 import pytest
 from streamsync.ss_types import EventRequest, InitSessionRequest, InitSessionRequestPayload, StreamsyncEvent
 import asyncio
+import contextlib
 
 from tests import test_app_dir
 
+
+@pytest.fixture
+def setup_app_runner():
```

Comment:
I would advocate to remove the dependence to pytest here, it's not required because we use the feature as a simple context manager. The boilerplate for pytest make that pretty heavy

```python
with setup_app_runner("./not_an_app", "run") as ar:
  pass
```

Reply:
```python
@contextlib.contextmanager
def setup_app_runner(app_dir: str, app_command: str):
    ar = AppRunner(app_dir, app_command)
    try:
        yield ar
    finally:
        ar.shut_down()
```

Reply:
```python
from conftest import setup_app_runner

def test_init_wrong_path(self) -> None:
    with setup_app_runner("./not_an_app", "run") as ar:
        with pytest.raises(SystemExit) as wrapped_e:
            ar.load()
        assert wrapped_e.type == SystemExit
        ```